src/frameworks/nuxt/templates/stores/__resources__.ts.ejs

```diff
@@ -0,0 +1,74 @@
+import { defineStore } from "pinia";
+import type { T<%= resourceNames.classify %>, T<%= resourceNames.classify %>Form } from "~/components/<%= resourceNames.plural %>/types";
+import { use<%= pluralize(resourceNames.classify) %>Service } from "~/services/<%= resourceNames.plural %>";
+
+export const use<%= pluralize(resourceNames.classify) %>Store = defineStore("<%= resourceNames.plural %>", () => {
+  // services
+  const { list, create, destroy, update } = use<%= pluralize(resourceNames.classify) %>Service();
+
+  // state
+  const <%= resourceNames.plural %> = ref<T<%= resourceNames.classify %>[]>();
```

Comment:
To have more consistency and to share more tools between the different frameworks, I intend to accept this PR, but I have plans to insert https://tanstack.com/query/latest/docs/framework/vue/overview here.